packages/storybook/stories/components/Textarea.stories.tsx

```diff
@@ -0,0 +1,173 @@
+/* @license CC0-1.0 */
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Textarea } from '@utrecht/component-library-react/src/css-module/Textarea';
+import readme from '@utrecht/components/textarea/README.md?raw';
+import tokensDefinition from '@utrecht/components/textarea/tokens.json';
+import tokens from '@utrecht/design-tokens/dist/index.json';
+import React from 'react';
+import { designTokenStory } from './util';
+
+const defaultArgs = {
+  dir: 'auto',
+  disabled: false,
+  focus: false,
+  hover: false,
+  invalid: false,
+  placeholder: undefined,
+  readOnly: false,
+  required: false,
+  spellCheck: false,
+  children: undefined,
+};
+
```

Comment:
The "Show code" has been affected, an unnecessary attribute `spellcheck` is included:

## Main branch

```html
<textarea class="utrecht-textarea utrecht-textarea--html-textarea" dir="auto">
  The Quick Brown Fox Jumps Over The Lazy Dog
</textarea>
```

## This branch

```html
<textarea
  spellcheck="false"
  class="utrecht-textarea utrecht-textarea--html-textarea"
  dir="auto"
></textarea>
```